fcrepo-auth-common/src/main/java/org/fcrepo/auth/common/FedoraUserSecurityContext.java

```diff
@@ -34,7 +32,7 @@
  * @author Gregory Jansen
  */
 public class FedoraUserSecurityContext implements SecurityContext,
-        AdvancedAuthorizationProvider {
+AdvancedAuthorizationProvider {
```

Comment:
Just to be clear, this is entirely a concern of @awoods. No one else has ever so much as noticed it.